README.md

```diff
@@ -1,22 +1,6 @@
-# Miami landing page
-- Implement [the landing page](https://www.figma.com/file/nHz8bflIwJaWP3P99vKTH5/miami_home_new?node-id=0%3A2)
-- Follow this [video guide](https://www.youtube.com/watch?v=sq184jq9DV8&feature=youtu.be) and write your own implementation.
+`Miami Condo Kings`
+First serious project, from Mate
```

Comment:
I guess it should describe the purpose of this web page (like that: "This is a web page for Miami Condo Kings company...")

Reply:
✓